.github/workflows/build.yml

```
@@ -54,6 +54,10 @@ jobs:
         run: |
           make lint
 
+      - name: Run type checker
+        run: |
+          make check
```

```suggestion
          npm run-script check
```

Sinon ça lance le check du makefile qui est autre chose (ou alors j'ai rien compris 😅)

---

Ah bien vu !